examples/preprocessing/plot_all_scaling.py

```diff
@@ -102,6 +102,8 @@
 # scale the output between 0 and 1 for the colorbar
 y = minmax_scale(y_full)
 
+# plasma does not exist in matplotlib < 1.5
+cmap = getattr(cm, 'plasma_r', cm.hot_r)
```

Comment:
For the colormaps, I tried to take one reasonably close from https://matplotlib.org/1.3.1/examples/color/colormaps_reference.html.